README.md

```diff
@@ -22,6 +22,15 @@ You will need to install it thanks to the `pip` binary.
 
 # Run the tests
 
+The tests can run both on the deployed backend and the mock backend. To run the complete
+test suite on both backend:
```

Comment:
You're talking about deployed/mock here but the everywhere you use local/remote. I'd be for sticking with local/remote and then update the wording across all repos at once.